src/main/kotlin/com/autonomousapps/internal/utils/gradleStrings.kt

```diff
@@ -206,11 +202,21 @@ internal fun Dependency.resolvedVersion(): String? = when (this) {
   else -> throw GradleException("Unknown Dependency subtype: \n$this\n${javaClass.name}")
 }?.intern()
 
-private fun Dependency.isJavaPlatform(): Boolean = when (this) {
-  is ProjectDependency -> dependencyProject.pluginManager.hasPlugin("java-platform")
+private fun Dependency.isDependencyToJavaPlatform(): Boolean = when (this) {
   is ModuleDependency -> {
     val category = attributes.getAttribute(Category.CATEGORY_ATTRIBUTE)
     category?.name == Category.REGULAR_PLATFORM || category?.name == Category.ENFORCED_PLATFORM
   }
   else -> throw GradleException("Unknown Dependency subtype: \n$this\n${javaClass.name}")
 }
+
+/**
+ * Return true if the dependency does not point at the "main" variant of the other module.
+ * Other variants are for example testFixtures() or Feature Variants with different capabilities.
+ */
+private fun Dependency.isDependencyToNonMainFeature(): Boolean = when (this) {
```

Comment:
Could you add links to the documentation for test fixtures and feature variants in your kdoc? I have a fairly verbose style when it comes to this documentation because the domain is so complex and it's easy to forget context as time passes.